pkg/operator/ceph/csi/predicate.go

```diff
@@ -48,12 +50,24 @@ func predicateController(ctx context.Context, c client.Client) predicate.Funcs {
 				if controller.DuplicateCephClusters(ctx, c, e.Object, false) {
```

Comment:
Will the csi controller never reconcile if there is more than one cluster? (unless the configmap exists?)

Reply:
It will never reconcile regardless of the presence of the configmap.